modules/microshift-firewall-known-issue.adoc

```diff
@@ -5,4 +5,5 @@
 :_content-type: CONCEPT
 [id="microshift-firewall-known-issue_{context}"]
 = Known firewall issue
+
 * To avoid breaking traffic flows with a firewall reload or restart, execute firewall commands before starting {product-title}. The CNI driver in {product-title} makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the CNI driver, but are deleted when the firewall reloads or restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after {product-title} is running, manually restart `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace to reset the rules controlled by the CNI driver.
```

Comment:
It seems a little odd to me to have a module that is just a single bullet point for a known issue. Typically, we leave known issues in the release notes so we don't have to scrub the docs when they are fixed. In some rare cases, an admonition might be added to the module where a customer might encounter the issue.